tests/unit/projection-test.js

```diff
@@ -1097,32 +1094,27 @@ module('unit/projection', function(hooks) {
                 name: PUBLISHER_NAME,
                 location: PUBLISHER_LOCATION,
                 owner: PUBLISHER_OWNER,
-              },
-              meta: {
-                projectionTypes: [PUBLISHER_CLASS, NORM_PROJECTED_PUBLISHER_CLASS]
               }
+            }, {
+              id: PUBLISHER_ID,
+              type: NORM_PROJECTED_PUBLISHER_CLASS,
+              attributes: {}
```

Comment:
The idea here is that the normalized payload would be roughly

```
{
  data: <TOP_LEVEL_PROJECTION>,
  included: [
    <PARTIAL_PRIMARY_RECORD>,
    <PARTIAL_RESOLVED_RECORD>,
    <RESOLVED_PROJECTION (empty)>
  ]
}
```

confirm?

Reply:
Actually, you are correct. I will check the changes again to ensure I haven't just placed the base record as a top level, when it should be in included.